radio/src/stamp.cpp

```diff
@@ -56,8 +56,8 @@ const char* getOtherVersion()
 }
 #elif defined(STM32)
 
-__attribute__ ((section(".fwversiondata"), used)) const char firmware_version[33] = "opentx-" FLAVOUR "-" VERSION " (" GIT_STR ")";
-__attribute__ ((section(".bootversiondata"), used)) const char boot_version[33] = "opentx-" FLAVOUR "-" VERSION " (" GIT_STR ")";
+__attribute__ ((section(".fwversiondata"), used)) const char firmware_version[] = "opentx-" FLAVOUR "-" VERSION " (" GIT_STR ")";
+__attribute__ ((section(".bootversiondata"), used)) const char boot_version[] = "opentx-" FLAVOUR "-" VERSION " (" GIT_STR ")";
```

Comment:
@schwabe didn't you assume some fixed size somewhere while reading that tag? I'm not sure anymore.

Reply:
Yeah, I was worried about that too, that's why I took the PR route

Reply:
@raphaelcoeffic I would assume so too. But everything seems fine.